osu.Game.Tests/Visual/Gameplay/TestSceneGameplaySamplePlayback.cs

```diff
@@ -20,14 +19,14 @@ public class TestSceneGameplaySamplePlayback : PlayerTestScene
         public void TestAllSamplesStopDuringSeek()
         {
             DrawableSlider slider = null;
-            DrawableSample[] samples = null;
+            PoolableSkinnableSample[] samples = null;
```

Comment:
any reason for the change here in type?

Reply:
Because the code below checks the value of `Playing`, but that has been moved to the channel now. I suppose I could add a `Playing` property on the sample itself, to indicate whether any channels are playing?

Reply:
Hmm, a bit confused. Internally `PoolableSkinnableSample` uses `DrawableSample` which has an `IsPlaying`. Does that not work any more?

Reply:
It's only `SkinnableSound` that has `IsPlaying`, which is an osu-only class that handles channels internally. It actually goes a bit deeper than just `IsPlaying`, for example TestSceneEditorSamplePlayback references `Looping`...

Reply:
Ah, I was on the wrong branch while checking. I see..